include/seqan3/io/sequence_file/format_sam.hpp

```diff
@@ -0,0 +1,280 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides the seqan3::sequence_file_format_sam class.
+ * \author Mitra Darvish <mitra.darvish AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <iterator>
+#include <string>
+#include <string_view>
+#include <vector>
+
+#include <range/v3/algorithm/copy.hpp>
```

Comment:
I think you do not need this header and the one below.
If you use `ranges::copy` anywhere change it to `std::ranges::copy`.